ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/worldstate/WorldStateStorage.java

```diff
@@ -28,6 +31,9 @@
 
   Optional<BytesValue> getAccountStorageTrieNode(Bytes32 nodeHash);
 
+  Optional<WorldStateProof<Bytes32, BytesValue>> getAccountProof(
```

Comment:
Can we actually move this method to `WorldStateArchive` so that the underlying storage class remains as simple as possible?